presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcClient.java

```diff
@@ -315,6 +315,16 @@ protected boolean filterSchema(String schemaName)
         }
     }
 
+    protected static Optional<Integer> getInteger(ResultSet resultSet, String columnLabel)
```

Comment:
We are not using `OptionalInt` very often. I know the interface is less rich than `Optional<Integer>`. Is that a reason?

Reply:
richness -- optionalint is warranted, when there is a benefit